test/controllers/markets_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class MarketsControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
So when you mentioned specs in your pull request message, I thought you meant tests! If you're not using all the files that get generated if you generate a controller (like this test file), you can always delete them. Having them hang around can *eventually* get confusing as an app grows.